applications/Octan_Network.md

```diff
@@ -0,0 +1,181 @@
+# Octan Network
+
+> This document will be part of the terms and conditions of your agreement and therefore needs to contain all the required information about the project. Don't remove any of the mandatory parts presented in bold letters or as headlines (except for the title)! Lines starting with a `>` (such as this one) should be removed. Please use markdown instead of HTML (e.g. `![](image.png)` instead of `<img>`). 
+>
+> See the [Grants Program Process](https://github.com/w3f/Grants-Program/#pencil-process) on how to submit a proposal.
+- **Team Name:** Octan Labs
+- **Payment Address:** Ethereum address (DAI) 0xC0e09A112Ae45d87597CD78c11b7D95a55aCC5F0
```

Comment:
We don't offer ERC20 payments anymore. Please refer to the readme and application template for payment options.

Reply:
Dear Semuelle, we committed new changes:

- Provide tech stacks for deliverables
- Add competitors about onchain reputation concept
- By Octan, "onchain reputation" means: (i) reputation scores are computed from onchain transactions; (ii) Web3 users mint Soulbound tokens to update and prove their reputation scores onchain.
- Changed ERC20 to Polkadot address
- Provided physical address
- Removed "scientific" attribute from research paper
- Removed "label to earn"